quarkus-picocli/src/test/java/PicocliIT.java

```diff
@@ -0,0 +1,118 @@
+import static java.util.concurrent.CompletableFuture.runAsync;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class PicocliIT {
+
+    static final String GREETING_COMMAND_DESC = "greeting";
+    static final String GREETING_NAME_OPTION = "--name";
+    static final String GREETING_NAME_VALUE = "QE";
+
+    static final String AGE_COMMAND_DESC = "age";
+    static final String AGE_OPTION = "--age";
+    static final String AGE_VALUE = "30";
+
+    @QuarkusApplication
+    static final RestService greetingApp = new RestService()
+            .withProperty("quarkus.args",
+                    GREETING_COMMAND_DESC + " " + GREETING_NAME_OPTION + " " + GREETING_NAME_VALUE)
```

Comment:
Here and everywhere in the file, using constants make the code both harder to read and longer. Maybe they should not be used?

Reply:
ok, I see your point here, I will change it for the commands Strings directly.